src/validation.h

```diff
@@ -200,10 +200,11 @@ void UnlinkPrunedFiles(const std::set<int>& setFilesToPrune);
 void PruneBlockFilesManual(int nManualPruneHeight);
 
 /** (try to) add transaction to memory pool
- * plTxnReplaced will be appended to with all transactions replaced from mempool **/
+ * plTxnReplaced will be appended to with all transactions replaced from mempool
+ * optionally takes an argument to return tx fee to the caller **/
```

Comment:
bcb85f4 perhaps document this added argument per https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments
```diff
- * optionally takes an argument to return tx fee to the caller **/
+ * @param[out] fee_out   optional argument to return tx fee to the caller **/
```

Reply:
done!